resources/idlharness.js

```diff
@@ -415,14 +417,14 @@ IdlArray.prototype.is_json_type = function(type)
                    if (thing.has_to_json_regular_operation()) { return true; }
                    var mixins = this.implements[thing.name];
                    if (mixins) {
-                       mixins = mixins.map(id => {
+                       mixins = mixins.map(function(id) {
                            var mixin = this.members[id];
                            if (!mixin) {
                                throw new Error("Interface " + id + " not found (implemented by " + thing.name + ")");
                            }
                            return mixin;
                        });
```

Comment:
I imagine you'll need to bind `this`, here, so: `mixins.map(function () { ... }, this)`